packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -151,27 +158,29 @@ const PersonalDetailsForm = props => {
                             />
                         )}
                         {'salutation' in values && (
-                            <RadioGroup
-                                className='dc-radio__input'
-                                name='salutation'
-                                selected={values.salutation}
-                                onToggle={e => {
-                                    e.persist();
-                                    setFieldValue('salutation', e.target.value);
-                                }}
-                                required
-                            >
-                                {salutation_list.map(item => (
-                                    <RadioGroup.Item
-                                        key={item.value}
-                                        label={item.label}
-                                        value={item.value}
-                                        disabled={
-                                            !!values.salutation && isFieldImmutable('salutation', editable_fields)
-                                        }
-                                    />
-                                ))}
-                            </RadioGroup>
+                            <span onClick={handleSalutationSelection}>
```

Comment:
Are we capturing every event within this span?
Will it be in sync with formik values as we've changed `selected` prop value

Reply:
thanks man. added the formik `setFieldValue` in click handler.